swagger.yaml

```diff
@@ -197,21 +190,26 @@ definitions:
       self:
         type: string
   Error:
```

Comment:
Could you use `Errors` (plural) like [here](https://github.com/osu-mist/express-api-skeleton/blob/master/openapi.yaml#L188-L217)? You can simply copy the `Error` and `Errors` objects from the express-api-skeleton.

Reply:
Done